docs/dimensions.md

```diff
@@ -42,9 +157,9 @@ Example: `var {height, width} = Dimensions.get('window');`
 
 **Parameters:**
 
-| Name      | Type     | Required | Description                                                                                   |
-| ------    | ------   | -------- | ----------------------------------------------------------------------------------------------|
-| dim       | string   | Yes      | Name of dimension as defined when calling `set`. @returns {Object?} Value for the dimension.  | 
+| Name | Type   | Required | Description                                                                                  |
+| ---- | ------ | -------- | -------------------------------------------------------------------------------------------- |
+| dim  | string | Yes      | Name of dimension as defined when calling `set`. @returns {Object?} Value for the dimension. |
```

Comment:
```suggestion
| dim  | string | Yes      | Name of dimension as defined when calling `set`. |

Returns an object with four number fields: `width`, `height`, `scale`, and `fontScale`.
```

`scale` and `numberScale` should maybe be documented better here, but honestly, I don't really know much about them!

Reply:
Indeed, these areas could use some more detail. I'll try them out and update the docs